goth/runner/__init__.py

```diff
@@ -203,8 +216,16 @@ def host_address(self) -> str:
         """Return the host IP address in the docker network used by the containers.
 
         Both the proxy server and the built-in web server are bound to this address.
+
+        On Mac (and Windows?) there's no network bridge and the services on the host
+        don't have access to Docker's internal network. Thus, we need to use a special
+        address `host.docker.internal`
         """
-        return self._compose_manager.network_gateway_address
+
+        if sys.platform == "linux":
+            return self._compose_manager.network_gateway_address
+        else:
+            return "host.docker.internal"
```

Comment:
Let's make this condition part of `ComposeManager#network_gateway_address`.

Reply:
well, actually, gateway address and host address are two different things ... and as far as I understand, the value here is needed for the runner and not necessarily for the compose component ... so, imvho, this condition belongs here and not in that other helper...

Reply:
Fair enough, let's leave it here.